doc/api/dns.md

```diff
@@ -353,15 +353,15 @@ be an object with the following properties:
 * `minttl`
 
 ```js
-{
+({
```

Comment:
In some places, I've added something like `const defaults = ;`, but it seems to me this does not fit everywhere. Maybe, somebody can think up a better approach.